### DISCUSSION TITLE
Getting a translation on top of the transcription

### DISCUSSION THREAD

**Discussion by mathdons** [openai/whisper#515] [Q&A] [answered]
Hello,

I'm looking for a way to also return a translation when doing a transcription. On the cli it's possible to **replace** the transcription by the translation, but I'd need both. Of course (once I figure out how to use that command directly in python...) I could do two passes, one to get the trancription and one to get the translation, but my use case is with 10s of thousands of lines, so processing time becomes an issue.

Thanks!

**Comment by mayeaux** [comment, marked as answer]
I had the same question: https://github.com/openai/whisper/discussions/420

My solution is to build an app that integrates with https://github.com/LibreTranslate/LibreTranslate and then use that to process translations after the transcription is completed.
